stdlib_test.go

```diff
@@ -251,11 +253,30 @@ func TestStdlibAdapterSubexps(t *testing.T) {
 			"msg":  ":.;<>_#{[]}\"\\",
 		},
 	} {
-		haveMap := subexps([]byte(input))
+		haveMap := subexps(StdlibAdapter{logRegexp: StdlibRegexpFull}, []byte(input))
 		for key, want := range wantMap {
 			if have := haveMap[key]; want != have {
 				t.Errorf("%q: %q: want %q, have %q", input, key, want, have)
 			}
 		}
 	}
 }
+
+func TestStdLibAdapterWithoutCaller(t *testing.T) {
+	buf := &bytes.Buffer{}
+	logger := NewLogfmtLogger(buf)
+	writer := NewStdlibAdapter(logger, Regexp(StdlibRegexpDefault))
+	for input, want := range map[string]string{
+		"error encoding and sending metric family: write tcp 127.0.0.1:9182->127.0.0.1:60125: wsasend:":                                     "msg=\"error encoding and sending metric family: write tcp 127.0.0.1:9182->127.0.0.1:60125: wsasend:\"\n",
+		"2023/04/28 07:28:46 error encoding and sending metric family: write tcp 127.0.0.1:9182->127.0.0.1:60125: wsasend:":                 "ts=\"2023/04/28 07:28:46\" msg=\"error encoding and sending metric family: write tcp 127.0.0.1:9182->127.0.0.1:60125: wsasend:\"\n",
+		"2023/04/28 07:28:46 /a/b/c/d.go:23: error encoding and sending metric family: write tcp 127.0.0.1:9182->127.0.0.1:60125: wsasend:": "ts=\"2023/04/28 07:28:46\" msg=\"/a/b/c/d.go:23: error encoding and sending metric family: write tcp 127.0.0.1:9182->127.0.0.1:60125: wsasend:\"\n",
+		"2009/01/23 01:23:23.123123 /a/b/c/d.go:23: hello":                                                                                  "ts=\"2009/01/23 01:23:23.123123\" msg=\"/a/b/c/d.go:23: hello\"\n",
+		"1:9182f": "msg=1:9182f\n",
```

Comment:
Use backtick-delimited strings to avoid the need to escape double-quotes. Also, maybe a good idea to use e.g. strings.TrimSpace to avoid needing to tack on a `\n` to the end of everything.